frontend/src/pages/org/crawls-list.ts

```diff
@@ -254,8 +254,8 @@ export class CrawlsList extends LiteElement {
 
   private renderControls() {
     return html`
-      <div class="grid grid-cols-12 gap-y-2 gap-x-4 items-center">
-        <div class="col-span-12 md:col-span-5">
+      <div class="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-[minmax(0,100%)_fit-content(100%)_fit-content(100%)] gap-x-2 gap-y-2 items-center">
+        <div class="row-start-1 col-span-1 md:col-span-2 lg:col-span-1">
```

Comment:
Why did you find that the `row-start-1` is needed here?

Reply:
Left over from previous change, isn't required, removed!